docs/data_requirements/input_data.md

```diff
@@ -71,11 +78,11 @@ It is likely that not every parameter concept is already covered by an ontology.
 It might, however, be indirectly possible by linking it to a distinct selection of related ontology concepts. 
 In this case you would link your parameter name to multiple related concepts.
 
-For a practical manual linking parameter names to multiple related ontology concepts, using MetaCreator or OAT, see [here](ontology.md#Link-a-parameter-name-to-multiple-related-ontology-concepts).
+For a practical manual linking parameter names to multiple related ontology concepts, using oemetadata builder, see [here](ontology.md#Link-a-parameter-name-to-multiple-related-ontology-concepts).
```

Comment:
Also a `here` link. The reason I suggest avoiding it is because whereas they are undestandable the first time one reads the document. I am sure people will refer to this document to get to some files. Having the names on the link helps by allowing the users to find what they want quickly

Reply:
I agree with what you say about the `here` links.